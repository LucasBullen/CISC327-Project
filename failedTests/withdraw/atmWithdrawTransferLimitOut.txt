2c2
< Logged into atm mode.
---
> Logged into agent mode.
4c4
< Enter amount to withdraw.
---
> Enter amount (in cents) to withdraw.
7,8c7,8
< Enter amount to withdraw.
< Session withdraw maximum exceeded. Withdraw cancelled.
---
> Enter amount (in cents) to withdraw.
> Withdraw above $1000.00 session limit. Withdraw cancelled.
