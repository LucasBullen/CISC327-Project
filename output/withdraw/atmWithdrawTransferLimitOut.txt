Select mode: atm or agent.
Logged into atm mode.
Enter account number to withdraw from.
Enter amount to withdraw.
Withdraw successful.
Enter account number to withdraw from.
Enter amount to withdraw.
Session withdraw maximum exceeded. Withdraw cancelled.
Logged out.
