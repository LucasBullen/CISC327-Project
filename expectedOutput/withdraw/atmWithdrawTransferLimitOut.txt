Select mode: atm or agent.
Logged into agent mode.
Enter account number to withdraw from.
Enter amount (in cents) to withdraw.
Withdraw successful.
Enter account number to withdraw from.
Enter amount (in cents) to withdraw.
Withdraw above $1000.00 session limit. Withdraw cancelled.
Logged out.
